app/models/edition.rb

```diff
@@ -659,7 +659,7 @@ def detailed_format
   end
 
   def content_store_document_type
-    PublishingApiPresenters.presenter_for(self).content.fetch(:document_type)
+    display_type_key
```

Comment:
we also seem to have `content_store_document_type` which calls `display_type_key`. I don't know if that's neater or just yet another piece of obfuscation.

Reply:
@bevanloon I'm confused about this, this is that method?

Reply:
bleach....yeah never mind I misread :( carry on.